pkg/resource/deploy/source_eval.go

```diff
@@ -1354,6 +1391,77 @@ func (rm *resmon) wrapTransformCallback(cb *pulumirpc.Callback) (TransformFuncti
 	}, nil
 }
 
+func (rm *resmon) wrapInvokeTransformCallback(cb *pulumirpc.Callback) (TransformInvokeFunction, error) {
```

Comment:
Can we document this function/explain why it exists/what the wrapping is exactly?